lib/ASTGen/Sources/ASTGen/ParameterClause.swift

```diff
@@ -83,7 +83,7 @@ extension ASTGenVisitor {
 
     let (secondName, secondNameLoc) = node.secondName.bridgedIdentifierAndSourceLoc(in: self)
 
-    var type = self.generate(optional: node.optionalType)
+    var type = node.optionalType.map(generate(type:))
```

Comment:
Do you need to change this to also use `generate(optional:)`?

Reply:
Now it returns `BridgedNullableTypeRepr` instead of `BridgedTypeRepr` which is not convenient to convert back to `BrigedTypeRepr`